inc/Engine/License/API/Pricing.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace WP_Rocket\Engine\License\API;
+
+class Pricing {
+	/**
+	 * The pricing data object
+	 *
+	 * @var object
+	 */
+	private $pricing;
+
+	/**
+	 * Instantiate the class
+	 *
+	 * @param object $pricing The pricing object.
+	 */
+	public function __construct( $pricing ) {
+		$this->pricing = $pricing;
+	}
+
+	/**
+	 * Gets the single license pricing data
+	 *
+	 * @return null|object
+	 */
+	public function get_single_pricing() {
+		return isset( $this->pricing->licenses->single ) ? $this->pricing->licenses->single : null;
+	}
+
+	/**
+	 * Gets the plus license pricing data
+	 *
+	 * @return null|object
+	 */
+	public function get_plus_pricing() {
+		return isset( $this->pricing->licenses->plus ) ? $this->pricing->licenses->plus : null;
```

Comment:
This method returns either `null` or an object. The code currently is checking if the property is set, but not that it has a value and its value is an object data type. Some ideas are:

```suggestion
		if ( ! isset( $this->pricing->licenses->plus ) || ! is_object( $this->pricing->licenses->plus ) ) {
			return null;
		}
		
		return $this->pricing->licenses->plus;
```